src/lib/Libnet/net_server.c

```diff
@@ -439,9 +439,10 @@ connection_idlecheck(void)
 static int
 engage_authentication(conn_t *pconn)
 {
-	int	ret;
-	int	sd;
-	char ebuf[ PBS_MAXHOSTNAME + 1 ];
+	int ret;
+	int sd;
+	char ebuf[PBS_MAXHOSTNAME + 1] = {'\0'};
+	char msgbuf[sizeof(ebuf) + 64];
```

Comment:
Yes, it's legal because the compiler is able to calculate the size at compile time. Appveyor builds are succeeding, so I don't think it's a problem for Windows.